basics/loops.md

```diff
@@ -470,7 +470,25 @@ while True:
         number = int(number)
     print(numbers)
     ```
-
+5. Make a programme to get a pyramid like shown in the OUTPUT section where user can type the number of rows needed
```

Comment:
Some nits that I would like to have fixed:
- "programme --> "program"
- "in the OUTPUT section" --> "below"

Reply:
- add `.` to the end